lib/css-converter.js

```diff
@@ -0,0 +1,311 @@
+import { CssSelectorParser } from 'css-selector-parser';
+import log from './logger';
+
+const CssConverter = {};
+
+const parser = new CssSelectorParser();
+parser.registerSelectorPseudos('has');
+parser.registerNestingOperators('>', '+', '~');
+parser.registerAttrEqualityMods('^', '$', '*', '~');
+parser.enableSubstitutes();
+
+const BOOLEAN_ATTRS = [
+  'checkable', 'checked', 'clickable', 'enabled', 'focusable',
+  'focused', 'long-clickable', 'scrollable', 'selected',
+];
+
+const NUMERIC_ATTRS = [
+  'index', 'instance',
+];
+
+const STR_ATTRS = [
+  'description', 'resource-id', 'text', 'class-name', 'package-name'
+];
+
+const ALL_ATTRS = [
+  ...BOOLEAN_ATTRS,
+  ...NUMERIC_ATTRS,
+  ...STR_ATTRS,
+];
+
+const attributeAliases = [
```

Comment:
ATTRIBUTE_ALIASES

Reply:
this could be a map

Reply:
I considered doing it that way, but then I would have had to do `_.toPairs(...)`, which would have translated it to this format. Defining it as a two-column matrix cuts out the middle-man.